themes/src/main/resources-community/theme/base/account/messages/messages_pt_BR.properties

```diff
@@ -4,15 +4,35 @@ doLogOutAllSessions=Sair de todas as sess\u00F5es
 doRemove=Remover
 doAdd=Adicionar
 doSignOut=Sair
+doLogIn=Entrar
+doLink=Vincular
+noAccessMessage=Acesso n\u00e3o permitido
```

Comment:
Does it belong here? Or it is a property for login messages?

Reply:
This string is also in the English file, but it doesn't seem to be used anywhere outside of login.